subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/dependency_verification.adoc

```diff
@@ -0,0 +1,849 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<verification-metadata>
+   <configuration>
+      <verify-metadata>true</verify-metadata>
+      <verify-signatures>false</verify-signatures>
+    </configuration>
+</verification-metadata>
+----
+
+Doing so, Gradle will verify all artifacts using <<#sec:checksum-verification,checksums>>, but will not verify <<#sec:signature-verification,signatures>>.
+Gradle will verify any artifact downloaded using its dependency management engine, which includes, but is not limited to:
+
+- artifact files (e.g jar files, zips, ...) used during a build
+- metadata artifacts (POM files, Ivy descriptors, Gradle Module Metadata)
+- plugins (both project and settings plugins)
+- artifacts resolved using the advanced dependency resolution APIs
+
+Gradle will _not_ verify changing dependencies (in particular `SNAPSHOT` dependencies) nor locally produced artifacts (typically jars produced during the build itself) as by nature their checksums and signatures would always change.
+
+With such a minimal configuration file, a project using _any_ external dependency or plugin would immediately start failing because it doesn't contain any checksum to verify.
+
+A dependency verification configuration is _global_: a single file is used to configure verification of the whole build.
+
+An easy way to get started is therefore to generate the minimal configuration for an existing build.
+
+[[sec:bootstrapping-verification]]
+=== Bootstrapping dependency verification
+
+It's worth mentioning that while Gradle can generate a dependency verification file for you, you should always check whatever Gradle generated for you because your build may _already_ contain compromised dependencies without you knowing about it.
+Please refer to the appropriate <<#sec:checksum-verification,checksum verification>> or <<#sec:signature-verification,signature verification>> section for more information.
+
+If you plan on using <<#sec:signature-verification,signature verification>>, please also read the <<sec:bootstrapping-signature-verification,corresponding section>> of the docs.
+
+Bootstrapping can either be used to create a file from the beginning, or also to _update_ an existing file with new information.
+Therefore, it's recommended to always use the same parameters once you started bootstrapping.
+
+The dependency verification file can be generated with the following CLI instructions:
+
+
+----
+gradle --write-verification-file sha256 help
+----
+
+The `write-verification-file` flag requires the list of <<#sec:checksum-verification,checksums>> that you want to generate or `pgp` for <<#sec:signature-verification,signatures>>.
+
+Executing this command line will cause Gradle to:
+
+* resolve all <<declaring_dependencies.adoc#sec:resolvable-consumable-configs,resolvable configurations>>, which includes:
+** configurations from the root project
+** configurations from all subprojects
+** configurations from `buildSrc`
+** included builds configurations
```

Comment:
It seems to be the only place that mentions `included builds`.

Can you please clarify what happens if:

1) A (uses verification) includes B (not using verification)
2) A (not using verification) includes B (uses verification) 

Does that mean Gradle would enforce everything to be verified? Is there a way to confine/prevent/ignore that?

included builds are quite useful for testing and developing Gradle projects and plugins, so iit would be sad to enforce strict verification.

What if A (uses verification) includes B (uses verification)? Which configuration file will be effectively used?